stable/spring-cloud-data-flow/README.md

```diff
@@ -1,16 +1,16 @@
 # Spring Cloud Data Flow Chart
 
-[Spring Cloud Data Flow](http://cloud.spring.io/spring-cloud-dataflow/) is a toolkit for building data integration and real-time data processing pipelines.
+[Spring Cloud Data Flow](https://cloud.spring.io/spring-cloud-dataflow/) is a toolkit for building data integration and real-time data processing pipelines.
```

Comment:
Let's update it to "A microservices-based Streaming and Batch data processing pipelines in Cloud Foundry and Kubernetes" for consistency in the messaging.